proto/src/determined/api/v1/api.proto

```diff
@@ -1834,9 +1834,20 @@ service Determined {
       get: "/api/v1/projects/{id}/columns"
     };
     option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
-      tags: "Internal"
+      tags: [ "Projects", "Internal" ]
     };
   }
+  // Get metrics range for a project.
+  rpc GetProjectMetricsRange(GetProjectMetricsRangeRequest)
```

Comment:
Maybe name this `GetProjectNumericMetricsRange`

to be explicit here that we aren't returning non numeric metrics